src/components/common/YourCompany/BusinessContactInfo.vue

```diff
@@ -5,9 +5,11 @@
     :originalBusinessContact="originalContact"
     :hasBusinessContactInfoChange="hasBusinessContactInfoChange"
     :editLabel="editLabel"
-    :editedLabel="editedLabel"
+    :editedLabel="editSavedLabel"
     :disableActions="isCorrectionFiling"
+    :disableActionTooltip="isChangeFiling"
     :invalidSection="invalidSection"
+    :optionalPhone="true"
```

Comment:
Did you try without the `:` ? Theoretically it should work since this is not a dynamic binding.

Reply:
I did! 
It works without the binding but i'm not sure the reason to write it differently than the other props.

Reply:
Sorry, it doesn't work how i would expect it too. Writing `optionalPhone` does not work, however assigning it a value does work `optionalPhone="true"`. 
That being said, you could declare it as `optionalPhone="fooBar"` and it will still work 😆

Reply:
I'm slightly leaning towards no colon just to show this isn't a bound prop, but your point is fair and I'm OK with the current code.

(Maybe -- though it may be optimized out by the compiler or Vue engine -- just maybe, this saves some reactive code from being attached to this prop, which isn't needed since this prop will never change. Back in the stone age, this was a valid manual optimization to save CPU cycles. Today this matters little. FWIW)

Reply:
For sure, thats a fair take. I do appreciate this insight!

Reply:
Updated this. 
Initially made it true because it's not editable in a correction but updated to be more explicit when we allow optional phone.

Reply:
> Sorry, it doesn't work how i would expect it too. Writing `optionalPhone` does not work, however assigning it a value does work `optionalPhone="true"`. That being said, you could declare it as `optionalPhone="fooBar"` and it will still work 😆

Yeah, I would expect just this prop's presence (no value) to work, as with other Vue props. I haven't figured that out yet.

"fooBar" probably works because it's truthy.